src/lib/tecs/TECS.hpp

```diff
@@ -272,6 +278,10 @@ class TECS
 	float _STE_rate_min{0.0f};					///< specific total energy rate lower limit acheived when throttle is at _throttle_setpoint_min (m**2/sec**3)
 	float _throttle_setpoint_max{0.0f};				///< normalised throttle upper limit
 	float _throttle_setpoint_min{0.0f};				///< normalised throttle lower limit
+	float _throttle_trim_min{0.0f};
+	float _throttle_trim{0.0f};					///< feedforward throttle applied when controller error is zero, is provided externally and can change in flight
```

Comment:
the `_throttle_trim` state doesnt change though, right? it gets adjusted for density, but the calibrated airspeed mapped value stays the same. And then the `_throttle_trim_mapped` is interpolated between the compensated values right? -- maybe a bit nitpicky .. but I think getting the naming consistent and understandable is important for these since we now have 3 trim values, 3 compensated values, and an interpolated value